src/Homework4/BrandNewGreenhouse.java

```diff
@@ -36,9 +183,10 @@ public static void main( String[] args ) {
         HousePlant fertilizeFicus = (HousePlant) ficus;
         fertilizeFicus.fertilizeHousePlant();
 
-        greenhouse1.buyPlant(ficus); //buy = add to greenhouse "Ficus"
-        greenhouse1.buyPlant(raspberry);//buy = add to greenhouse "Raspberry"
-        greenhouse1.buyPlant(tulip);//buy = add to greenhouse "Raspberry"
+        // add plants to greenhouse
+        //greenhouse1.buyPlant(ficus);
```

Comment:
Please don't leave commented code in branches, that's a bad practice

Reply:
Deleted commented code.